tests/func/test_checkout.py

```diff
@@ -880,3 +880,31 @@ def test_checkout_executable(tmp_dir, dvc):
         assert not isexec
     else:
         assert isexec
+
+
+def test_checkout_partial(tmp_dir, dvc):
+    tmp_dir.dvc_gen(
+        {"db": {"a.txt": "a", "b.txt": "b", "sub_dir": {"c.txt": "c"}}}
+    )
+
+    db_dir = tmp_dir / "db"
+    sub_dir = db_dir / "sub_dir"
+    file_1, file_2, sub_dir_file = [
+        db_dir / filename for filename in ("a.txt", "b.txt", sub_dir / "c.txt")
+    ]
+
+    shutil.rmtree(db_dir)
+
+    dvc.checkout(targets=[str(file_1)])
```

Comment:
*Nitpick*: `checkout` supports a single `str` argument as well. 
```suggestion
    dvc.checkout(targets=str(file_1))
```

Reply:
ah, cool, thanks!